.github/workflows/e2e.yaml

```diff
@@ -37,5 +37,5 @@ jobs:
                 working-directory: ${{ matrix.directory }}
 
             -
-                run: ./../../bin/rector process src --dry-run --ansi
+                run: ./../../bin/rector process --dry-run --ansi
```

Comment:
`src` path is needed as `e2e/template-extends` doesn't has path config in the rector.php

Reply:
with this PR the path-config is added.

I dropped it here, because IMO the tests should be isolated and the github action workflow should not make assumptions about the tests file structure